### PR TITLE
Fix op repr

### PR DESCRIPTION
**Pull request by serpilliere**

### CODE REVIEW

File: miasm2/expression/expression.py [hunk 1]
```diff
@@ -793,11 +791,13 @@ def __init__(self, op, *args):
     def __str__(self):
         if self.is_associative():
             return '(' + self._op.join([str(arg) for arg in self._args]) + ')'
+        if (self._op.startswith('call') or
```
Comment (commial) [inline miasm2/expression/expression.py]: To mym ind, `call_func_` is more precize for your catching

File: miasm2/expression/expression.py [hunk 2]
```diff
@@ -624,7 +622,7 @@ def visit(self, cb, tv=None):
         src2 = self._src2.visit(cb, tv)
         if (cond == self._cond and
             src1 == self._src1 and
-            src2 == self._src2):
+                src2 == self._src2):
```
Comment (fmonjalet) [inline miasm2/expression/expression.py]: As this line is indented, maybe the previous could also be.

File: miasm2/expression/expression.py [hunk 3]
```diff
@@ -793,11 +791,13 @@ def __init__(self, op, *args):
     def __str__(self):
         if self.is_associative():
             return '(' + self._op.join([str(arg) for arg in self._args]) + ')'
+        if (self._op.startswith('call_func_') or
+            len(self._args) > 2 or
+                self._op in ['parity', 'segm']):
```
Comment (fmonjalet) [inline miasm2/expression/expression.py]: Same remark here (indentation).

File: miasm2/expression/expression.py (hunk 2 above)
Comment (serpilliere) [inline miasm2/expression/expression.py]: It's generated by autopep8...

File: miasm2/expression/expression.py (hunk 3 above)
Comment (serpilliere) [inline miasm2/expression/expression.py]: Autopep8 as well

File: miasm2/expression/expression.py (hunk 2 above)
Comment (fmonjalet) [inline miasm2/expression/expression.py]: autopep8 is mad... :(

**Pull request merged**